src/hackerspace_online/tests/test_management_commands.py

```diff
@@ -85,3 +89,25 @@ def test_initdb(self):
             self.assertTrue(Site.objects.exists())
 
             Tenant.objects.get(schema_name=apps.get_app_config('library').TENANT_NAME)  # no assert, but will throw exception if doesn't exist
+
+
+class GenerateContentTest(TenantTestCase, CommandMixin):
+    """ generate adds items to an existing tenant.
```

Comment:
generate adds...typo?

Reply:
was using generate as a shorthand for generate_content. fixed